File: Binary to Decimal/binary_to_decimal.py
#WAP in Python to find the decimal number of a given binary number

binary_num = list(input("Input a binary number: "))
decimal_num = 0

for i in range(len(binary_num)):
	digit = binary_num.pop()
	if digit == '1':
		decimal_num = decimal_num + pow(2, i)
print("The decimal value of the number is", decimal_num)

File: Binary to Decimal/binary_to_decimal.c
#include<stdio.h>
int main()
{
    int binary_num, decimal_num=0,last_digit;
    int power=1;

    printf("Binary number  : ");
    scanf("%d",&binary_num);

    while(binary_num!=0)
    {
        last_digit = binary_num%10;
        decimal_num = decimal_num + (last_digit*power);
        binary_num = binary_num/10;
        power = power*2;
    }

    printf("Decimal Number : %d",decimal_num);
    return 0;
}